prow/plugins/verify-owners/verify-owners.go

```diff
@@ -126,27 +169,56 @@ func handle(ghc githubClient, gc *git.Client, log *logrus.Entry, pre *github.Pul
 		}
 	}
 
-	// Check each OWNERS file.
+	// If OWNERS_ALIASES file exists, get all aliases.
+	// If the file was modified, check for non trusted users in the newly added owners.
+	nonTrustedUsers, repoAliases, err := nonTrustedUsersInOwnersAliases(ghc, log, triggerConfig, org, repo, r.Dir, modifiedOwnerAliasesFile.Patch, ownerAliasesModified, skipTrustedUserCheck)
+	if err != nil {
+		return err
+	}
+
+	// Check if OWNERS files have the correct config and if they do,
+	// check if all newly added owners are trusted users.
+	pr, err := ghc.GetPullRequest(org, repo, number)
+	if err != nil {
+		return fmt.Errorf("error loading PullRequest: %v", err)
+	}
+	oc, err := roc.LoadRepoOwners(org, repo, pr.Base.Ref)
```

Comment:
I've found it hard to mentally unpack what `oc` is - original-config...?

Reply:
`oc` is ownersClient - https://github.com/kubernetes/test-infra/pull/12460/files/7f27a3184a8ef331762222932444adfb6699e7e8#diff-f6224796619ee4a96a9ed4d3d25fa27aR67